domains/zak1o7.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "zak1o7",
+           "email": "",
+           "discord": "821192904735195138",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.qNiLIiLQ6eZ9djIug-7g7YCgwo1hX_kQJ5urGrsIu2zZ28H3hNtFZmKyRlEzK8oyY5rY9zDSsSsTAEx2jMWHtbCNWNXb-MWcj3A-DLdswQ3_FrpgnAjEInkfOS1Ga3ttc5iKVED_PF4v350RibPsHfRo7dpEkuue3r2cqt0MpJoTfeCjABf0DFljiGShfzJYIagBONCg4rQ1iwykV5Y7s0_KmVFfTwA0suJYDmjHUXdLlmlyN7nG1BLnPxsTEYXM-FAwyFckZrCOMPBROMfx5fPy0Jp42tktbEQaBkg53mq5dbHts6_p2pdl-MLZmqJej36uvQvcT-_1jRv2GgKpSA.alQE6XnMjdqdi2zJhDyaKg.vuCq17zfCjWoK9XF2gZdoYtUzOu_YqoXAKGtfObZkW-JkatyEVbPFpVOk0eAEPXMchdzr9wNCXMlcvBugoGMpQ.RvT8c3BqufKNHQFml5kUpQ"
+        },
+    
+        "record": {
+            "CNAME": "zak1o7.github.io"
```

Comment:
You should complete your website first, I only see a "Hello, World!"

Reply:
> You should complete your website first, I only see a "Hello, World!"

ah alright man, thanks for the reply tho. Can I request again after I completely build the website? and also after registering can I also edit my Website again?

Reply:
> ah alright man, thanks for the reply tho. Can I request again after I completely build the website? and also after registering can I also edit my Website again?

yes you can, take your time we don't mind— open a new pr once you're ready